lib/runner/job.ex

```diff
@@ -125,11 +120,26 @@ defmodule ElixirBench.Runner.Job do
       network_mode: @network_mode,
       image: "elixirbench/runner:#{job.config.elixir_version}-#{job.config.erlang_version}",
       volumes: ["#{get_benchmars_output_path(job)}:#{container_benchmars_output_path}:Z"],
-      depends_on: deps,
-      environment: build_runner_environment(job)
+      depends_on: Map.keys(deps),
+      environment: build_runner_environment(job),
+      command: build_runner_command_from_deps(deps)
     }
   end
 
+  defp build_runner_command_from_deps(deps) do
+    command =
+      Enum.reduce(deps, "", fn {_dep_name, dep_params}, acc ->
```

Comment:
I think renaming `acc` to something like `command` here helps the readability as it makes it more obvious the accumulator is hopening the command or maybe the `wait_command` something like that

Reply:
fixed! :heavy_check_mark: